app/controllers/group_members_controller.rb

```diff
@@ -0,0 +1,72 @@
+class GroupMembersController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    group = Group.find(params[:group_id])
+
+    res = current_user &&
+      (group.is_admin?(current_user) || group.is_mod?(current_user))
+    members = res ? group.members : group.members.accepted
+    members = members.page(params[:page]).per(20)
+    render json: members, meta: {cursor: 1 + (params[:page] || 1).to_i}
+  end
+
+  def create
+    membership_hash = params.require(:group_member).permit(:group_id, :user_id).to_h
+    user = User.find(membership_hash['user_id'])
+
+    return error! "Wrong user", 403 if user.nil? || current_user != user
```

Comment:
Idk, it just felt like an odd behavior to silently convert any random user ID to the current user, instead of throwing an error